app/components/mandatarissen/mandataris-edit-prompt.hbs

```diff
@@ -2,16 +2,32 @@
   <div class="au-o-grid__item au-u-1-2">
     <AuContent @skin="small">
       <AuHeading @level="4" @skin="5">Wijziging huidige situatie</AuHeading>
-      <p>Indien de mandataris wordt vervangen, of de huidige
-        <strong>status, fractie, rangorde of bevoegdheid</strong>
-        verandert:
-      </p>
-      <ul>
-        <li>beëindig dit mandaat door de einddatum in te vullen;</li>
-        <li><strong>start een nieuw mandaat</strong>
-          met de gewijzigde situatie.</li>
-      </ul>
-      <AuButton {{on "click" this.changeStatus}}>Verander status</AuButton>
+      {{#if this.mandaatHasEnded}}
+        <p>
+          Dit mandaat werd beëindigd en kan niet meer gewijzigd worden. Indien
+          deze status niet correct is, gebruik de 'corrigeer fouten'
+          functionaliteit hiernaast.
+        </p>
+        <p>
+          Indien er een nieuw mandaat toegevoegd moet worden, bijvoorbeeld na
+          een periode waarin het mandaat niet actief was, voeg een nieuw mandaat
+          toe.
+        </p>
+      {{else}}
+        <p>Indien de mandataris wordt vervangen, of de huidige
+          <strong>status, fractie, rangorde of bevoegdheid</strong>
+          verandert:
+        </p>
+        <ul>
+          <li>beëindig dit mandaat door de einddatum in te vullen;</li>
+          <li><strong>start een nieuw mandaat</strong>
+            met de gewijzigde situatie.</li>
```

Comment:
The text asks to fill in the end date to end the mandate and then start a new mandate, but when I click on "Verander status" there is no option to fill in the end date. This might be confusing to the user.